lib/src/list_multimap/list_multimap_builder.dart

```diff
@@ -12,10 +12,9 @@ part of built_collection.multimap;
 /// [Built Collection library documentation](#built_collection/built_collection)
 /// for the general properties of Built Collections.
 class ListMultimapBuilder<K, V> {
-  bool _copyBeforeWrite;
-  BuiltListMultimap<K, V> _builtListMultimap;
   Map<K, BuiltList<V>> _builtMap;
-  Map<K, ListBuilder<V>> _builderMap = new Map<K, ListBuilder<V>>();
+  BuiltListMultimap<K, V> _builtMapOwner;
```

Comment:
I like this better, but maybe there's a better option still. We can discuss at length later :)